### PR TITLE
Fix Lint/HandleExceptions

### PR DESCRIPTION
**Pull request by jaysonesmith**
## Details

- Rubocop doesn't like it when you block hole exceptions. Since we're not wanting to error out on the load error here, I've just added a brief warn message.

## Motivation and Context

Working to help solve issue [1021](https://github.com/cucumber/cucumber-ruby/issues/1021)!

## How Has This Been Tested?

`bundle exec rake` :+1:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Rubocop style fixes

### CODE REVIEW

**Comment by jaysonesmith** [comment]
This one definitely needs to get squashed. Ended up basing it off of the todoregen branch erroneously and now it's a hot mess but should still be good to go overall.

**Comment by jaysonesmith** [comment]
Will work to fix conflicts today if I can!

File: lib/cucumber/rake/task.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
   # Support Rake > 0.8.7
   require 'rake/dsl_definition'
 rescue LoadError
+  warn 'LoadError for rake/dsl_definition'
```
Comment (olleolleolle) [inline lib/cucumber/rake/task.rb]: This is one of those place where we _want_ to silence the LoadError and not caring about it.

For ancient Rake versions before that version, this file won't exist (and won't be necessary) and Ruby will raise the LoadError. We catch that and silence that error, telling Ruby: "This is normal."

So, we want to silence RuboCop's complaining about this specific LoadError silencing.

Comment (jaysonesmith) [reply lib/cucumber/rake/task.rb]: I've pushed a new version without this. :)

**Pull request merged**